django_project/base/management/commands/geocontext.json

```diff
@@ -71,47 +71,121 @@
+        },
+        {
+            "key": "erf",
+            "name": "Erven Boundaries",
+            "description": null,
+            "url": "http://maps.kartoza.com/geoserver",
+            "query_type": "WFS",
+            "result_regex": "qgs:id",
+            "time_to_live": 604800,
+            "srid": 4326,
+            "layer_typename": "kartoza:erf",
+            "service_version": "1.0.0"
+        },
+        {
+            "key": "district_municipalities",
+            "name": "District Municipalities",
+            "description": null,
+            "url": "http://maps.kartoza.com/geoserver",
+            "query_type": "WFS",
+            "result_regex": "qgs:district",
+            "time_to_live": 604800,
+            "srid": 3857,
+            "layer_typename": "kartoza:district_municipalities",
+            "service_version": "1.0.0"
+        },
+        {
+            "key": "municipal_ward_boundaries",
+            "name": "Municipal Ward Boundaries",
+            "description": null,
+            "url": "http://maps.kartoza.com/geoserver",
+            "query_type": "WFS",
+            "result_regex": "qgs:mn_name",
+            "time_to_live": 604800,
+            "srid": 3857,
+            "layer_typename": "kartoza:municipal_ward_boundaries",
+            "service_version": "1.0.0"
+        },
+        {
+            "key": "sub_places",
+            "name": "Sub Places",
+            "description": null,
+            "url": "http://maps.kartoza.com/geoserver",
+            "query_type": "WFS",
+            "result_regex": "qgs:sp_name",
+            "time_to_live": 604800,
+            "srid": 4326,
+            "layer_typename": "kartoza:sub_places",
+            "service_version": "1.0.0"
         }
     ],
     "context_group": [
         {
-            "key": "context_group_key_1",
-            "name": "Context Group Name 1",
-            "description": "Aloha make way make way",
+            "key": "river_catchments_group",
+            "name": "River Catchments",
+            "description": "Layers that relate to freshwater,estuarine and maritime ecosystems",
             "context_service_registry_keys": [
-                "provinces",
                 "water_management_area",
-                "primary_catchment_area"
+                "primary_catchment_area",
+                "secondary_catchment_area",
+                "tertiary_catchment_area",
+                "quaternary_catchment_area"
             ]
         },
         {
-            "key": "megazord_group",
-            "name": "Megazord",
+            "key": "political_boundary_group",
+            "name": "Administrative Boundary",
             "description": "Power Rangers",
             "context_service_registry_keys": [
-                "secondary_catchment_area",
-                "quaternary_catchment_area",
-                "provinces"
+                "provinces",
+                "kartoza:district_municipalities",
```

Comment:
Hi @NyakudyaA this should be `district_municipalities`, see line 100 (https://github.com/ismailsunni/geocontext/pull/1/files#diff-454ce8e0f350c869eecd5e16ce6c61a1R100)

Use the key, not the `layer_typename`.

Can you check the rest also, since I saw there are many of them.